doc/ref_model/chapters/chapter08.rst

```diff
@@ -659,6 +659,10 @@ operations and management complexity.
 At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with
 different form factors.
 
+Finally, there is a need to integrate, and to consider extreme-edge devices which can be part of the ecosystem,
```

Comment:
As discussed in the RM meeting of 2022-12-07, starting the sentence with "Finally" doesn't follow the previous sentences. If this suggestion is accepted, then lines 663 and 664 should be deleted. Orchestration is not the topic of this section and so is being removed.

If orchestration must be mentioned, then it should be a separate paragraph that applies to the content of this section.
```suggestion
Additionally, the extreme edge may consist of programmable devices as part of the
Telco Edge Cloud ecosystem for use in meeting some extremely low latency requirements,
```